addons/hr_holidays/data/hr_holidays_data.xml

```diff
@@ -1,55 +1,311 @@
+            <field name="public" eval="True"/>
+            <field name="type">url</field>
+            <field name="url">/hr_holidays/static/src/img/icons/Training_Time_Off_2.svg</field>
+        </record>
+        <record id="icon_28" model="ir.attachment">
+            <field name="name">Unpaid_Time_Off.svg</field>
+            <field name="res_model">hr.leave.type</field>
+            <field name="res_field">icon</field>
+            <field name="public" eval="True"/>
+            <field name="type">url</field>
+            <field name="url">/hr_holidays/static/src/img/icons/Unpaid_Time_Off.svg</field>
+        </record>
+        <record id="icon_29" model="ir.attachment">
+            <field name="name">Unpaid_Time_Off_2.svg</field>
+            <field name="res_model">hr.leave.type</field>
+            <field name="res_field">icon</field>
+            <field name="public" eval="True"/>
+            <field name="type">url</field>
+            <field name="url">/hr_holidays/static/src/img/icons/Unpaid_Time_Off_2.svg</field>
+        </record>
+        <record id="icon_30" model="ir.attachment">
+            <field name="name">Work_Accident_Time_Off.svg</field>
+            <field name="res_model">hr.leave.type</field>
+            <field name="res_field">icon</field>
+            <field name="public" eval="True"/>
+            <field name="type">url</field>
+            <field name="url">/hr_holidays/static/src/img/icons/Work_Accident_Time_Off.svg</field>
+        </record>
+        <record id="icon_31" model="ir.attachment">
+            <field name="name">Work_Accident_Time_Off_2.svg</field>
+            <field name="res_model">hr.leave.type</field>
+            <field name="res_field">icon</field>
+            <field name="public" eval="True"/>
+            <field name="type">url</field>
+            <field name="url">/hr_holidays/static/src/img/icons/Work_Accident_Time_Off_2.svg</field>
+        </record>
+    </data>
     <data noupdate="1">
         <!-- Casual leave -->
         <record id="holiday_status_cl" model="hr.leave.type">
             <field name="name">Paid Time Off</field>
-            <field name="allocation_type">fixed</field>
+            <field name="requires_allocation">yes</field>
+            <field name="employee_requests">no</field>
             <field name="leave_validation_type">both</field>
-            <field name="allocation_validation_type">both</field>
-            <field name="color_name">black</field>
+            <field name="allocation_validation_type">officer</field>
+            <!-- <field name="color_name">black</field> -->
             <field name="leave_notif_subtype_id" ref="mt_leave"/>
             <field name="allocation_notif_subtype_id" ref="mt_leave_allocation"/>
-            <field name="validity_start" eval="time.strftime('%Y-%m-01')"/>
             <field name="responsible_id" ref="base.user_admin"/>
+            <field name="icon" ref="hr_holidays.icon_14"/>
```

Comment:
We could use something like 
```xml
            <field name="image_1920" type="base64" file="base/static/img/res_partner_address_3.jpg"/>
```
instead of creating ir.attachments right ?